src/background/services/background.ts

```diff
@@ -179,6 +168,17 @@ export class Background {
         await this.storage.populate()
         await this.openPaymentsService.genererateKeys()
       }
+      await this.checkPermissions()
     })
   }
+
+  checkPermissions = async () => {
```

Comment:
For consistency:
```suggestion
  async checkPermissions() {
```

Reply:
Would need to `.bind(this)` at call-site this way.